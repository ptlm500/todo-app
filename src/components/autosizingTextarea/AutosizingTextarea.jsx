import styled from 'styled-components';
import TextareaAutosize from 'react-autosize-textarea';
import { theme } from '../../styles';

export default styled(TextareaAutosize)`
  background: ${theme.background};
  color: ${theme.text};
  font: inherit;
  appearance: none;
  border: none;
  border: 1px solid ${theme.divider};
  padding: 0px;
  width: 100%;
`;
