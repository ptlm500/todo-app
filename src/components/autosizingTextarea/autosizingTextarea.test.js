import React from 'react';
import renderer from 'react-test-renderer';
import AutosizingTextarea from './AutosizingTextarea';
import 'jest-styled-components';

describe('AutosizingTextarea', () => {
  it('renders an AutosizingTextarea', () => {
    const tree = renderer.create(<AutosizingTextarea />);
    expect(tree).toMatchSnapshot();
  });
});
