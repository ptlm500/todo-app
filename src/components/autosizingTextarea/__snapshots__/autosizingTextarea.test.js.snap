// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutosizingTextarea renders an AutosizingTextarea 1`] = `
.c0 {
  background: #FFFFFF;
  color: #212121;
  font: inherit;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  border: none;
  border: 1px solid #BDBDBD;
  padding: 0px;
  width: 100%;
  max-width: 100%;
}

<textarea
  async={false}
  className="c0"
  onChange={[Function]}
  rows={1}
/>
`;
